### PR TITLE
CHECK-169: BE: Create seeder for testing clock in

### PR DESCRIPTION
**Pull request by JayJayAquino**
# Description

BE: Create seeder for testing clock in

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: database/factories/TimeLogFactory.php [hunk 1]
```diff
@@ -25,19 +25,45 @@
...
     $timeSheet = TimeSheet::where('organization_id', $orgId)->first();
 
-    $date =  "2019-02-01";
-    $time = "06:30:44";
+
+    // Generate a random date with ranges
+    // upper-limit -> today, lower-limit -> today-6months
+    $tempDate = Carbon::now()->subMonths(rand(0,6))->subDays(rand(0,7))->subHours(rand(0, 12));
+
+    $year = $tempDate->year;
```
Comment (404tonynotfound) [inline database/factories/TimeLogFactory.php:38]: if the tempdate object has the same property names as the ones we need why we doing this?

File: database/factories/TimeLogFactory.php [hunk 2]
```diff
@@ -25,19 +25,45 @@
...
+    $time = $hour.':'.$minute.':'.$second;
+
     $clockIn = new TimeStamp($date, $time);
 
     $carbonClockIn = $clockIn->carbon;
 
-    $date =  "2019-02-01";
-    $time = "09:30:44";
+    // Simulate a shift
+    $tempDate = $tempDate->addHours(rand(1,8));
```
Comment (404tonynotfound) [inline database/factories/TimeLogFactory.php:54]: how can you guarantee this wont be earlier than the last random time creation?

**Review by 404tonynotfound — APPROVED** [review]

File: database/factories/TimeLogFactory.php (hunk 2 above)
Comment (ChanM0) [reply database/factories/TimeLogFactory.php:54]: the add hours ensures its later

**Review by ChanM0 — APPROVED** [review]
👍

File: database/factories/TimeLogFactory.php (hunk 1 above)
Comment (JayJayAquino) [reply database/factories/TimeLogFactory.php:38]: I think I could rename it to `$date`

**Review by chippingcolt — APPROVED** [review]

**Pull request merged**